src/modules/popup.js

```diff
@@ -0,0 +1,81 @@
+import displayComments from './displayCmds.js';
+import addComment from './addCmd.js';
+
+const popupSection = document.createElement('section');
+popupSection.setAttribute('id', 'popupModal');
+popupSection.classList.add('modal');
+popupSection.innerHTML = `
+<div class="popup-wrapper">
+    <button class="close-btn">&times;</button>
+    <img class="popup-img" alt="">
+    <h2>Artwork</h2>
+    <div class="art-details">
+        <ul>
+            <li>Paint: Water Paint</li>
+            <li>sizes: 18 x 24 inches</li>
+            <li>Made: German</li>
+            <li>Sale at: 999999$</li> 
+        </ul>
+    </div>
+    <div class="cmds">
+      <h3 id="c-count">Comments(<span id="c-total"></span>)</h3>
+      <ul id="all-comments"></ul>
+    </div>
+    <div class="cmd-form">
+        <form action="#" class="form">
+            <input type="text" name="name" id="name" placeholder="Your Name">
+            <input type="text" name="insight" id="insights" placeholder="Your insights">
```

Comment:
## Suggestion Changes
This change is minor but will help improve your app. I know there was a lot to be done and some areas tend to skip off our minds when working on deadlines.

- [ ] What if you could add a basic HTML validation to these input fields so empty data don't get posted? You could pass the required attribute to the input fields so that only when these fields have data before they get posted 👍🏽

```html
<input required type="text" name="name" id="name" placeholder="Your Name">
<input required type="text" name="insight" id="insights" placeholder="Your insights">
```